### DISCUSSION TITLE
How to get event emitter for uploader

### DISCUSSION THREAD

**Discussion by anli-xsigns** [quasarframework/quasar#11560] [General - Components / Directives / etc] [answered]
Hello,
I use QUploader to upload some files. After upload I want to remove the uploaded files from the queue. I try to use the removeUploadedFiles() method in @uploaded Event. However, I'm unable to get the reference to the QUploader Instance directly in the event (e. g. like event.target on some vue events). How can I get the correct reference to the uploader?

**Comment by MilosPaunovic** [comment, marked as answer]
[https://codepen.io/milospaunovic/pen/rNGOPJY?editors=1011
](https://codepen.io/milospaunovic/pen/rNGOPJY?editors=1011
)
You should use `refs`, read more about 'em [here](https://v3.vuejs.org/guide/composition-api-template-refs.html#template-refs).
